src/intervals/trigonometric.jl

```diff
@@ -2,24 +2,24 @@
 
 
 # hard code this for efficiency:
-const one_over_half_pi_interval = inv(0.5 * pi_interval(Float64))
+const one_over_half_pi_interval = inv(0.5 * Interval{Float64}(π))
 
 """
 Multiply an interval by a positive constant.
 For efficiency, does not check that the constant is positive.
 """
 multiply_by_positive_constant(α, x::Interval) = @round(α*x.lo, α*x.hi)
 
-half_pi(::Type{Float64}) = multiply_by_positive_constant(0.5, pi_interval(Float64))
-half_pi(::Type{T}) where {T} = 0.5 * pi_interval(T)
+half_pi(::Type{Float64}) = multiply_by_positive_constant(0.5, Interval{Float64}(π))
+half_pi(::Type{T}) where {T} = 0.5 * Interval{T}(π)
```

Comment:
Naive question: would it help here to use `multiply_by_positive_constant`? (Same question to other lines below)